inference/enterprise/parallel/README.md

```diff
@@ -0,0 +1,31 @@
+# Parallel Robofolow Inference server
+
+Introducing the highly concurrent implementation of Roboflow Inference Server! This version of the server accepts and processes requests asynchronously, running the web server, preprocessing, auto batching, inference, and post processing all in separate threads to increase server fps throughput. Separate requests to the same model will be batched on the fly as allowed by `$MAX_BATCH_SIZE`, and then response handling will occurr independently. Images are passed via python's SharedMemory module to maximize throughput.
```

Comment:
We should add a note here about the enterprise licensing.

Reply:
moved it up in the readme